ui/i18n/datepicker-ko.js

```diff
@@ -25,7 +25,7 @@ datepicker.regional.ko = {
 	dayNamesShort: [ "일","월","화","수","목","금","토" ],
 	dayNamesMin: [ "일","월","화","수","목","금","토" ],
 	weekHeader: "주",
-	dateFormat: "yy-mm-dd",
+	dateFormat: "yy. mm. dd.",
```

Comment:
The month and day should be single digit when possible. I'm not sure which form of the year we would use here. @rxaviers, do we display the medium or short date from CLDR?